### PR TITLE
Use git_version instead of vergen

### PR DESCRIPTION
**Pull request by klochowicz**
This popular crate uses `git` CLI instead of compiling whole libgit.
Output version is just as good.

The only minus is that we need `git` installed in order to compile the project.

### CODE REVIEW

**Comment by bonomat** [comment]
> The only minus is that we need git installed in order to compile the project.

I think this is acceptable 😅

File: vergen-version/src/lib.rs [hunk 1]
```diff
@@ -1,8 +1,11 @@
 /// Return git semver version
```
Comment (bonomat) [inline vergen-version/src/lib.rs:1]: I think we can rename this module to `version` or similar.

Comment (klochowicz) [reply vergen-version/src/lib.rs:1]: yes, indeed! just wanted to submit as draft first to see whether this actually gives us anything!

**Comment by klochowicz** [comment]
hm, for some reason the crate is not being cached locally and it wants to rebuild it for the smoke-test... this is pretty weird. 🤔

**Comment by klochowicz** [comment]
> hm, for some reason the crate is not being cached locally and it wants to rebuild it for the smoke-test... this is pretty weird. 🤔

 it must be invaliding the cache between runs, because it depends on something external (`git describe` command). This is a bit unfortunate; unless such a crate is at the very top of the dependency pile, it will cause recompilation of other dependent crates; it will likely not result in any speedup :/ I need to doublecheck whether `identify` protocol sends daemon version; if it does, then this PR is likely to worsen the performance further.

**Comment by klochowicz** [comment]
Closing this PR as cache invalidation makes this unfeasible as a performance improvement, we might as well stick to vergen.

**Pull request closed without merging**